next-docs/pages/cli/config.mdx

```diff
@@ -1 +1,21 @@
 # Config
+
+To install the MUD command line interface (CLI) into your environment, run this command:
+
+```sh copy
+pnpm install -g @latticexyz/cli@next
+```
+
+Alternatively, you can install `@latticexyz/cli` as part of a Node project, in which case you can access it from the project's root using either `pnpm mud` or `./node_modules/.bin/mud`
```

Comment:
This is the recommended approach since the version needs to align with the project. Wonder if we should just remove the global installation to avoid confusion with misaligned versions.

Reply:
Did you push the change? I still see the global installation instructions here (line 5-7)

Reply:
To be clear: the recommended way of using the MUD cli is to install it as a dev dependency in the project and then run via `pnpm mud` from the project root